Gemfile

```diff
@@ -34,8 +34,5 @@ end
 
 group :test do
   gem "codecov", require: false
-  # Workaround for cc-test-reporter with SimpleCov 0.18.
-  # Stop upgrading SimpleCov until the following issue will be resolved.
-  # https://github.com/codeclimate/test-reporter/issues/418
-  gem "simplecov", "~> 0.10", "< 0.18"
+  gem "simplecov", "~> 0.18.5"
```

Comment:
Tests are passing but there's an issue pushing the coverage to code climate. Could it be related to this? The mentioned issue is closed but maybe the fix wasn't published yet, I'll look into it.

Reply:
yes, we should ditch codeclimate to control codecoverage, I'll make a PR using codecov instead. We can skip this for now